prow/config.yaml

```diff
@@ -19654,6 +19663,14 @@ periodics:
       secret:
         secretName: ssh-key-secret
         defaultMode: 0400
+    - name: cache-ssd
+      hostPath:
+        path: /mnt/disks/ssd0
+    - name: docker-graph
+      hostPath:
+        path: /mnt/disks/ssd0/docker-graph
+    - name: var-lib-docker
```

Comment:
we can drop this, we explicitly relocate the docker graph to /docker-graph now in the image. I have no plans to change this since it just works